components/gorgias_oauth/common/override-app.mjs

```diff
@@ -0,0 +1,17 @@
+import app from "../gorgias_oauth.app.mjs";
+const appName = "gorgias";
+
+function overrideApp(base) {
```

Comment:
Do you mind to add a comment here just explaining the idea of this function. I think it will help the maintainability in the future